doc/release/2021-08-releases.rst

```diff
@@ -49,28 +52,140 @@ Core
+    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`5385`)
+
+-   Now streams and interactive transactions over streams are implemented in iproto.
+    Stream is associated with its ID, which is unique within one connection.
+    All requests with the same, not zero stream ID, belong to the same stream.
+    All requests in the stream are processed synchronously.
+    The execution of the next request will not start until the previous one is completed.
+    If a request has zero stream ID, it does not belong to a stream and is processed in the old way.
+
+    In ``net.box`, a stream is an object above the connection that has the same methods
+    but allows to execute requests sequentially. ID is generated on the client side automatically.
+    If a user writes his own connector and wants to use streams, he must transmit ``stream_id`` over the iproto protocol.
+
+    The primary purpose of streams is transactions via iproto.
+    Each stream can start its transaction, so they allow multiplexing several transactions over one connection.
+    There are multiple ways to begin, commit and rollback a transaction.
+    One can do that using appropriate stream methods, using ``call`` or ``eval`` methods,
+    or the ``execute`` method with SQL transaction syntax. User can mix these methods.
+    For example, start a transaction using ``stream:begin()``,
+    and commit transaction using ``stream:call('box.commit')`` or ``stream:execute('COMMIT')``.
+    If any request fails during the transaction, it will not affect the other requests in the transaction.
+    If a disconnect occurs when there is some active transaction in the stream,
+    this transaction will be rollbacked if it does not have time to commit before this moment.
+
+-   Add new ``memtx_allocator`` option to ``box.cfg{}``.
+    It allows selecting the appropriate allocator for memtx tuples if necessary.
+    Possible values are ``system`` for malloc allocator and ``small`` for default small allocator.
+
+    Implement system allocator, based on malloc: slab allocator, used for tuples allocation,
+    has a particular disadvantage --- it tends to unresolvable fragmentation on specific workloads (size migration).
+    In this case, a user should be able to choose another allocator.
+    System allocator is based on malloc function but restricted by the same quota as a slab allocator.
+    The system allocator does not alloc all memory at the start. Instead, it allocates memory as needed,
+    checking that quota is not exceeded
+    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`5419`).
+
+Replication
+~~~~~~~~~~~
+
+-   Introduce ``box.info.replication[n].downstream.lag`` field to monitor the state of replication.
+    This member represents a lag between a moment when the main node writes a certain transaction to its own WAL
+    and a moment it receives an ack for this transaction from a replica
+    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`5447`).
```

Comment:
```suggestion
    This represents the lag between the moment when the main node writes a certain transaction to its WAL
    and the moment it receives an ack for this transaction from the replica
    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`5447`).
```
I understand the lag value is independent for each replica, so I wrote "the replica" as "the replica in question"